src/Symfony/Component/Finder/Iterator/VcsIgnoredFilterIterator.php

```diff
@@ -73,4 +85,13 @@ private function readGitignoreFile(string $path): ?string
 
         return $this->gitignoreFilesCache[$path] = Gitignore::toRegex(file_get_contents($path));
     }
+
+    private function normalizePath(string $path): string
+    {
+        if ('\\' === \DIRECTORY_SEPARATOR) {
+            return str_replace('\\', '/', $path);
```

Comment:
will this work with UNC Win paths like `\\server/dir`?

Reply:
I assume it will. Not sure this can be easily proven with a test though.